src/urllib3/contrib/securetransport.py

```diff
@@ -500,31 +523,33 @@ def handshake(
                 result = Security.SSLHandshake(self.context)
 
                 if result == SecurityConst.errSSLWouldBlock:
-                    raise socket.timeout("handshake timed out")
+                    raise socket.timeout("handshake timed out")  # type: ignore
```

Comment:
Curious about this one. Mypy doesn't like `socket.timeout`?

Reply:
`src/urllib3/contrib/securetransport.py:526: error: Argument 1 to "timeout" has incompatible type "str"; expected "int"`